lightning/src/ln/channelmanager.rs

```diff
@@ -6358,13 +6357,37 @@ where
 		}
 	}
 
+	/// Returns whether we have pending HTLC forwards that need to be processed via
+	/// [`Self::process_pending_htlc_forwards`].
+	pub fn is_pending_htlc_processing(&self) -> bool {
```

Comment:
s/is_pending_htlc_processing/needs_pending_htlc_processing? Right now to me it's ambiguous whether it means we're currently processing htlcs.

Reply:
Yeah same, I agree with the reasoning against `has_pending_htlc_forwards` though.